Sources/RInAppMessaging/Router.swift

```diff
@@ -110,11 +109,15 @@ internal class Router: RouterType, ViewListenerObserver {
             return
         }
 
+        guard let presenter = getPresenter(for: campaignViewType) else {
```

Comment:
If we move this inside `displayQueue.async {` the assertionFailure() will be handled inside displayQueue.async block and cannot be tested.
Also, I refer to how's `displayTooltip` works when finding **presenter** first before it can go through the below code.